DiscordBot/Models/Signup.cs

```diff
@@ -4,5 +4,11 @@ class Signup
     {
         public string Id;
         public string Role;
```

Comment:
You need to add a getter here, otherwise they can be publically changed by default 🙂